src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -94,25 +101,26 @@
     LoadingCache<Long, LogUnitEntry> dataCache;
     long maxCacheSize;
 
-    private final AbstractLocalLog localLog;
+    private AbstractLocalLog localLog;
+
+    public static long maxLogFileSize = Integer.MAX_VALUE;  // 2GB by default
```

Comment:
haven't actually seen this used in this file. is this being declared as a global variable for cmdlet use?

I guess it is (saw later). Should probably turn this into a constant, upper case name.